index.html

```diff
@@ -13,5 +13,41 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Acme Web Sulutions</title>
   </head>
-  <body></body>
+  <body>
+    <header class="site-header">
+      <div class="site-wrapper">
+        <div>header</div>
+      </div>
+    </header>
+    <section class="first-section">
+      <div class="site-wrapper">
+        <div>1</div>
+      </div>
+    </section>
+    <section class="second-section">
+      <div class="site-wrapper">
+        <ul class="column-container">
+          <li class="column-item">1</li>
+          <li class="column-item">2</li>
+          <li class="column-item">3</li>
+        </ul>
+      </div>
+    </section>
+    <div class="site-wrapper">
+      <section class="third-section">1</section>
+    </div>
+    <div class="site-wrapper">
+      <ul class="fourth-section">
```

Comment:
Here is better to use two sections for them instead of ul. They arent a list. They are two separate sections.

Reply:
Done.

Reply:
Haven't you pushed it? I still see the ul and lis.

Reply:
I pushed it.